src/DurableTask.Netherite/StorageLayer/Faster/LogWorker.cs

```diff
@@ -123,12 +126,30 @@ void AddToFasterLog(byte[] bytes)
                     bool isLastFragment = 1 + bytes.Length - pos <= this.maxFragmentSize;
                     int packetSize = isLastFragment ? 1 + bytes.Length - pos : this.maxFragmentSize;
                     bytes[pos - 1] = (byte)(((pos == 1) ? first : none) | (isLastFragment ? last : none));
-                    this.log.Enqueue(new ReadOnlySpan<byte>(bytes, pos - 1, packetSize));
+                    this.AppendToLog(new ReadOnlySpan<byte>(bytes, pos - 1, packetSize));
                     pos += packetSize - 1;
                 }
             }
         }
 
+        void AppendToLog(ReadOnlySpan<byte> span)
+        {
+            this.log.Enqueue(span);
+            if (this.traceLogDetails)
+            {
+                this.TraceLogDetail("Appended", this.log.TailAddress, span);
+            }
+        }
```

Comment:
wait so these are the literal bytes appended to the FASTER log? Could this contain private data?

Reply:
The `TraceLogDetail` method is not tracing the actual contents, just information about it.